typescript/infra/scripts/check-deploy.ts

```diff
@@ -1,14 +1,21 @@
 import { HyperlaneCore, HyperlaneCoreChecker } from '@hyperlane-xyz/sdk';
```

Comment:
Is that `Submodule helloworld deleted from 646f1e` diff block above true or just submodule weirdness?